py-denormalized/Cargo.toml

```diff
@@ -10,7 +10,7 @@ name = "denormalized_python"
 crate-type = ["cdylib"]
 
 [dependencies]
-pyo3 = { version = "0.21.2", features = ["experimental-async"] }
+pyo3 = { workspace = true }
```

Comment:
did we need pyo3 out of the py-denormalized crate?

Reply:
yes it did. `cargo update` doesn't work otherwise because it goes down a circular resolver path.